tools/torchfix/torchfix/torchfix.py

```diff
@@ -9,12 +9,21 @@
 )
 
 from .visitors.performance import TorchSynchronizedDataLoaderVisitor
+from .visitors.misc import TorchRequireGradVisitor
 
 __version__ = "0.0.2"
 
 DEPRECATED_CONFIG_PATH = Path(__file__).absolute().parent / "deprecated_symbols.yaml"
 
 
+def GET_ALL_VISITORS():
+    return [
+        TorchDeprecatedSymbolsVisitor(DEPRECATED_CONFIG_PATH),
+        TorchRequireGradVisitor(),
+        TorchSynchronizedDataLoaderVisitor(),
+    ]
```

Comment:
Minor nit here but does this need to be a function? Wouldn't a global variable just be better here?

```suggestion
ALL_VISITORS = [
    TorchDeprecatedSymbolsVisitor(DEPRECATED_CONFIG_PATH),
    TorchRequireGradVisitor(),
    TorchSynchronizedDataLoaderVisitor(),
]
```

Reply:
I need to different copies for TorchChecker and TorchCodemod, otherwise the tests will fails because the state will be changed after the first one.

Reply:
Oh I see because you modify the variable?